src/pages/ProductListing/ProductListing.jsx

```diff
@@ -1,6 +1,7 @@
 import "./ProductListing.css";
 import { FilterBar } from "../../components/FilterBar/FilterBar";
 import { ProductsByFilter } from "../../components/ProductsByFilter/ProductsByFilter";
+
```

Comment:
Remove this

Reply:
yes auto import forgot to delete it will fix now